pytentiostat/routines.py

```diff
@@ -38,16 +39,20 @@ def _initialize_arduino(com):
     Creates board object with Arduino(). If the connection fails it prints an error message and exits.
     Parameters
     ----------
-    com: string
         the COM port that the potentiostat is connected to.
     """
     try:
         board = Arduino(com,
                         baudrate=_BAUD_RATE)  # opens communication to Arduino
-        print("Pytentiostat connected to {}.\n".format(
-            com))
+
+
     except:
-        sys.exit("Error. Could not open COM port")
+        sys.exit("ERROR: Could not open port Arduino Uno is connected to. Exiting…")
+
+    if board.firmware == 'pytentiostat_firmata.ino':  # check if the board have the right firmware
```

Comment:
On my system, `board.firmware` returns "pytent_firm.ino". Not sure why it's different.

Reply:
I don't sure but maybe you could have changed the name of the firmware uploaded in the arduino? I used the one uploaded here https://github.com/juami/potentiostat_firmata.

Reply:
Oh I see. The name must have changed when we made that repository. I will update mine.